### ISSUE TITLE
Color

### ISSUE THREAD

**Issue by TCNOco** [GeorgLegato/Txt2Vectorgraphics#3]
This may be out of scope for this project, may not be... Assuming I generate cartoons, is there a similar tool for creating SVG images with coloured shapes? I assume a lot of compressing colours and gradients down to solid shapes would need to be done, but even generating something "South Park"-inspired, for example.

It this tool currently able to do something like this?

**Comment by GeorgLegato** [comment]
I have a preview already implemented, but too bad quality... i post it in few mins on reddit

**Comment by TCNOco** [comment]
Yeah, I see potrace is [B&W only](https://superuser.com/questions/1544218/how-to-retain-color-when-using-potrace), but I have seen a few tools online that convert (relatively well) images to color SVGs. Don't know of anything that can be locally run/called.

Take [Adobe's image to SVG](https://www.adobe.com/express/feature/image/convert/svg) for example.

**Comment by GeorgLegato** [comment]
https://www.reddit.com/r/StableDiffusion/comments/xtu3e9/preview_colored_icons_as_txt2vectorgraphics/

Adobe struggles as well on the halftones in picture:
![image](https://user-images.githubusercontent.com/7210708/193467068-bb54b7dc-0684-46a8-a42d-f2d8eae47626.png)

**Comment by GeorgLegato** [comment]
something like that i guess. check the results are too pickled and noise inside the svg (Kim Kardashain in southpark)

txt2png:
![download](https://user-images.githubusercontent.com/7210708/193468270-ed4625c6-c13c-43e0-9cd7-6b031e2642af.png)

svg:
![00001-3395043740](https://user-images.githubusercontent.com/7210708/193468287-5dbdc44c-e844-48f7-8920-9f24ff2b02e9.svg)

**Comment by VexedPanda** [comment]
Looking close at the vtracer demos on their site, it’s evident there too. It’s introducing patches of a third colour in between boundaries of two colours.

**Comment by GeorgLegato** [comment]
btw, vectormagic.com has exactly the solution as needed. they produce really perfect vectors on colored/antialiased input.
closed source ,-)

**Comment by some9000** [comment]
Also really looked into finding something that could do colored tracing here. No usable results. And yeah, Vector Magic is the best vectorizer by far. Around 2010 they had a desktop app. If you look really hard, you can probably find a usable version. Not sure about licensing, though.
